<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pandore Region Merging Interface</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>380</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Open images to enable options</string>
      </property>
      <property name="title">
       <string>Region Merging Options</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Algorithm:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Number of merging:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Threshold:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QComboBox" name="cbAlgorithm">
               <item>
                <property name="text">
                 <string>pcontrastmerging</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>pentropymerging</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>pmeanmerging</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>puniformitymerging</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>pvariancemerging</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>pboundarymerging</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>phistomerging</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="helpForAlgorithm">
               <property name="toolTip">
                <string>Online help</string>
               </property>
               <property name="text">
                <string>?</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QSlider" name="valueNumber">
               <property name="toolTip">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;number&lt;/span&gt; specifies the number of allowed merging. If &lt;span style=&quot; font-style:italic;&quot;&gt;number&lt;/span&gt; = -1 then all possible merging are done.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="minimum">
                <number>-1</number>
               </property>
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="value">
                <number>-1</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdNumber">
               <property name="numDigits">
                <number>4</number>
               </property>
               <property name="segmentStyle">
                <enum>QLCDNumber::Filled</enum>
               </property>
               <property name="value" stdset="0">
                <double>-1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QSlider" name="valueThreshold">
               <property name="toolTip">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;threshold&lt;/span&gt; specifies the maximum difference allowed between two regions to decide to merge them.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="value">
                <number>10</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdThreshold">
               <property name="numDigits">
                <number>4</number>
               </property>
               <property name="segmentStyle">
                <enum>QLCDNumber::Filled</enum>
               </property>
               <property name="value" stdset="0">
                <double>10.000000000000000</double>
               </property>
               <property name="intValue" stdset="0">
                <number>10</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Display</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QRadioButton" name="optShowOryginalImage">
            <property name="text">
             <string>Oryginal image</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="optShowImageForComputations">
            <property name="text">
             <string>Image for computations</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QRadioButton" name="optShowLabeledImage">
                <property name="text">
                 <string>Labeled image</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="optShowColorLabeledImage">
                <property name="text">
                 <string>Color labeled image</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="optShowImageWithTransparentLabels">
                <property name="text">
                 <string>Image with transparent labels</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="optShowImageWithLabelBorder">
                <property name="text">
                 <string>Image with border of labels</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QToolButton" name="bttInvertLabelValue">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Invert</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="bttRandomColorForLabel">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Random</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="valueTransparentMask">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.050000000000000</double>
                </property>
                <property name="value">
                 <double>0.150000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="borderColorChooser">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Set color</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>44</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>521</width>
         <height>533</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>919</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_oryginal_image"/>
    <addaction name="actionOpen_image_for_computation"/>
    <addaction name="actionOpen_prelabeled_image"/>
    <addaction name="separator"/>
    <addaction name="actionSave_result"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Pandore"/>
    <addaction name="actionAbout_this_GUI"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_oryginal_image">
   <property name="text">
    <string>Open oryginal image</string>
   </property>
  </action>
  <action name="actionOpen_image_for_computation">
   <property name="text">
    <string>Open image for computation</string>
   </property>
  </action>
  <action name="actionOpen_prelabeled_image">
   <property name="text">
    <string>Open prelabeled image</string>
   </property>
  </action>
  <action name="actionLoad_test_images">
   <property name="text">
    <string>Load test images</string>
   </property>
   <property name="toolTip">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;table style=&quot;-qt-table-type: root; margin-top:4px; margin-bottom:4px; margin-left:4px; margin-right:4px;&quot;&gt;
&lt;tr&gt;
&lt;td style=&quot;border: none;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;ala &lt;span style=&quot; font-weight:600;&quot;&gt;ma&lt;/span&gt; kota&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout_Pandore">
   <property name="text">
    <string>About Pandore</string>
   </property>
  </action>
  <action name="actionAbout_this_GUI">
   <property name="text">
    <string>About this GUI</string>
   </property>
  </action>
  <action name="actionSave_result">
   <property name="text">
    <string>Save result</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>cbAlgorithm</tabstop>
  <tabstop>helpForAlgorithm</tabstop>
  <tabstop>valueNumber</tabstop>
  <tabstop>valueThreshold</tabstop>
  <tabstop>optShowOryginalImage</tabstop>
  <tabstop>optShowImageForComputations</tabstop>
  <tabstop>optShowLabeledImage</tabstop>
  <tabstop>bttInvertLabelValue</tabstop>
  <tabstop>optShowColorLabeledImage</tabstop>
  <tabstop>bttRandomColorForLabel</tabstop>
  <tabstop>optShowImageWithTransparentLabels</tabstop>
  <tabstop>valueTransparentMask</tabstop>
  <tabstop>optShowImageWithLabelBorder</tabstop>
  <tabstop>borderColorChooser</tabstop>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>valueNumber</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>optShowLabeledImage</sender>
   <signal>toggled(bool)</signal>
   <receiver>bttInvertLabelValue</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>optShowColorLabeledImage</sender>
   <signal>toggled(bool)</signal>
   <receiver>bttRandomColorForLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>optShowImageWithTransparentLabels</sender>
   <signal>toggled(bool)</signal>
   <receiver>valueTransparentMask</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>optShowImageWithLabelBorder</sender>
   <signal>toggled(bool)</signal>
   <receiver>borderColorChooser</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
